src/explore-education-statistics-admin/src/services/userService.ts

```diff
@@ -43,18 +55,17 @@ export interface Role {
   normalizedName: string;
 }
 
-export interface ReleaseRole {
-  name: string;
-  value: string;
+export interface ResourceRoles {
+  Publication: [] | undefined;
```

Comment:
Similar to the other comments you should be able to do: `Publication?: []`

Reply:
Sorted now :)